tests/foreman/cli/test_computeresource.py

```diff
@@ -169,14 +134,14 @@ def test_delete(self):
          u'description': gen_string('alphanumeric', 255)},
         {u'name': gen_string(str_type='utf8'),
          u'description': gen_string(str_type='utf8')},
-        {u'name': '<html>%s</html>' %
-                  gen_string(str_type='alpha'),
-         u'description': '<html>%s</html>' %
-                         gen_string(str_type='alpha')},
-        {u'name': "%s[]@#$%%^&*(),./?\"{}><|''" %
-                  gen_string(str_type='utf8'),
-         u'description': "%s[]@#$%%^&*(),./?\"{}><|''" %
-                         gen_string(str_type='alpha')},
+        {u'name': u'<html>{0}</html>'.format(
+            gen_string(str_type='alpha')),
+         u'description': u'<html>{0}</html>'.format(
+             gen_string(str_type='alpha'))},
+        {u'name': u"{0}[]@#$%^&*(),./?\\\"{{}}><|''".format(
+            gen_string(str_type='utf8')),
+         u'description': u"{0}[]@#$%^&*(),./?\\\"{{}}><|''".format(
+             gen_string(str_type='alpha'))},
```

Comment:
nice formatting change. not a blocker, but any reason for keeping the str_type= in the argument?